lib/App/index.js

```diff
@@ -711,7 +668,26 @@ class App {
       });
 
 		})
-	}
+  }
+  
+  async _getPackFileEntries(prunePaths) {
+    const walker = new ignoreWalk.Walker({
+      path: this.path,
+      ignoreFiles: [ '.homeyignore', '' ],
```

Comment:
Is there a reason for this empty string in the `ignoreFiles`?

Reply:
It's part of the thing I mentioned on slack, about trying to contact the original developer of the library.It doesn't have a built-in way to provide some default ignore rules that should be used. This is basically a hack to work around that 😦 

The `onReadIgnoreFile` call allows me to provide some default ignore rules, without actually making a file on the file system. That "file" has a filename of empty string, to prevent undesired side effects. If I gave it a name that could exist on the file system, someone would be able to create such a file and add some rules to it.

Anyway, the `onReadIgnoreFile` by itself doesn't do anything without also defining the order in which rules are applied. This line provides the right order: default rules last, so `.homeyignore` cannot override the rules.

There's some discussion here: https://github.com/npm/ignore-walk/issues/2, but the author has so far ignored the feature request.

Reply:
I see, thanks for the explanation! These sorts of things might be valuable to add as a comment to the code.